src/pages/in/embed/_id/button.vue

```diff
@@ -296,6 +295,14 @@ export default {
         console.error(err); // eslint-disable-line no-console
       }
     },
+    onClickLoginButton() {
+      window.open(
+        `https://${LIKE_CO_HOSTNAME}/in/register/api`,
```

Comment:
@mckingho think this might need some `referrer`/`querystring` change

Reply:
Not related